docs/2.0/reference/pipelines/configurations-as-code/api.md

```diff
@@ -105,6 +105,27 @@ repository {
 </HclListItemExample>
 </HclListItem>
 
+### `env` block
+
+<HclListItem name="env" requirement="optional" type="block">
+<HclListItemDescription>
+
+Env blocks are configuration components used by [repository](#repository-blocks) blocks to specify environment variables that will be set when executing Terragrunt commands. The block contains a map of environment variable names and their values.
+
+</HclListItemDescription>
+<HclListItemExample>
+
+```hcl
+repository {
+  env {
+    TF_VAR_environment = "an_environment_provided_input"
+    TG_LOG_FORMAT         = "json"
```

Comment:
This one is also dangerous... as we're overriding the log format in pipelines now ;/

Reply:
Switched to less problematic examples